.travis.yml

```diff
@@ -29,7 +29,7 @@ deploy:
   provider: script
   script: release-manager --config .release.yml --check-version --make-version --make-artifact --upload-artifact
   on:
-    condition: '"${TRAVIS_GO_VERSION}" == "1.7.4"' 
+    condition: '"${TRAVIS_GO_VERSION}" == "1.9.2"'
     tags: true
 
```

Comment:
^^ These changes belong in #112

Reply:
But it's travis.yml. You wrote about it in #110.  Did I understand you wrong?

Reply:
Delete the rows in one commit, add the new rows in the other commit.